runtime/runtime/src/lib.rs

```diff
@@ -1388,6 +1398,7 @@ impl Runtime {
                 protocol_version,
             )
             .map_err(RuntimeError::StorageError)?;
+        total.add(gas_used_for_migrations, gas_used_for_migrations)?;
```

Comment:
Moving this here is not a no-op. The TotalGuard implements drop which will now get called with different data for missing chunks. I'll defer to the runtime folks to judge if this is fine or not since I have no clue what are migrations.